plugin/trino-memory/src/test/java/io/trino/plugin/memory/TestMemoryConnectorTest.java

```diff
@@ -164,6 +164,28 @@ public void testCustomMetricsScanOnly()
         assertThat(((Count) metrics.getMetrics().get("finished")).getTotal()).isGreaterThan(0);
     }
 
+    @Test
+    public void testExplainCustomMetricsScanOnly()
+    {
+        DistributedQueryRunner runner = (DistributedQueryRunner) getQueryRunner();
+        MaterializedResult result = runner.execute(
+                getSession(),
+                "EXPLAIN ANALYZE VERBOSE SELECT partkey FROM part");
+        String explainAnalyze = (String) result.getOnlyValue();
```

Comment:
`BaseConnectorTest` extends `AbstractTestQueryFramework` which has a bunch of utility methods to simplify this. The generic way is
```java
String result = (String) computeActual("EXPLAIN ...").getOnlyValue();
```

The simpler version for this case is
```java
String result = (String) computeScalar("EXPLAIN ...");
```

There's actually a more specific `assertExplainAnalyze` method that you could use here if you wanted to convert your assertion into a regex. (we should probably refactor that method to return `AbstractStringAssert<?>` so you can do your own assertions on the result...)

Reply:
Thanks, TIL about `assertExplainAnalyze` :)